lib/services/android-project-service.ts

```diff
@@ -311,6 +290,10 @@ class AndroidProjectService implements IPlatformProjectService {
 			}
 		}
 
+		// metadata generation support
+		args = args.concat(["-Dns.resources", path.join(__dirname, "../../resources/tools")]);
+		args = args.concat((["-Dns.project.libs", this.platformData.libDir]));
```

Comment:
From custom_rules.xml we don't know where resources are installed, and i need to use the metadata-generator.jar  in the custom ant step